src/iomei.cpp

```diff
@@ -7666,6 +7690,35 @@ bool MEIInput::ReadAnnot(Object *parent, pugi::xml_node annot)
     }
 }
 
+bool MEIInput::ReadAnnotScore(Object *parent, pugi::xml_node annot)
+{
+    AnnotScore *vrvAnnotScore = new AnnotScore();
+    // this->ReadEditorialElement(annot, vrvAnnotScore);
+
+    vrvAnnotScore->ReadPlist(annot);
+    // vrvAnnotScore->ReadSource(annot);
+
+    parent->AddChild(vrvAnnotScore);
+    // vrvAnnotScore->m_content.reset();
+
+    bool hasNonTextContent = false;
+    // copy all the nodes inside into the document
+    for (pugi::xml_node child = annot.first_child(); child; child = child.next_sibling()) {
+        const std::string nodeName = child.name();
+        if (!hasNonTextContent && (!nodeName.empty())) hasNonTextContent = true;
+        // vrvAnnotScore->m_content.append_copy(child);
+    }
+    this->ReadUnsupportedAttr(annot, vrvAnnotScore);
+    // Unless annot has only text we do not load children because they are preserved in Annot::m_content
+    if (hasNonTextContent) {
+        return true;
+    }
+    else {
+        // vrvAnnot->m_content.remove_children();
+        return this->ReadTextChildren(vrvAnnotScore, annot, vrvAnnotScore);
+    }
```

Comment:
Keep only what you use now and remove the commented code. We can easily put things back whenever needed.

Reply:
I've taken cancelled code out of here, but left a few comments to remind myself what's gone